addons/website_event_booth/static/src/js/booth_register.js

```diff
@@ -196,37 +207,33 @@ publicWidget.registry.boothRegistration = publicWidget.Widget.extend({
         ev.preventDefault();
         ev.stopPropagation();
 
-        $(ev.currentTarget).addClass('disabled').attr('disabled', 'disabled');
-
-        const $form = this.$('#o_wbooth_contact_details_form');
-        if (this._isConfirmationFormValid($form)) {
-            const formData = new FormData($form[0]);
-            const response = await $.ajax({
-                url: `/event/${encodeURIComponent(this.$el.data('eventId'))}/booth/confirm`,
-                data: formData,
-                processData: false,
-                contentType: false,
-                type: 'POST',
-            });
+        ev.currentTarget.classList.add("disabled");
+        ev.currentTarget.disabled = true;
 
-            const jsonResponse = response && JSON.parse(response);
+        const formEl = this.el.querySelector("#o_wbooth_contact_details_form");
+        if (this._isConfirmationFormValid(formEl)) {
+            const formData = new FormData(formEl);
+            const jsonResponse = await post(`/event/${encodeURIComponent(this.el.dataset.eventId)}/booth/confirm`, formData);
             if (jsonResponse.success) {
                 this.el.querySelector('.o_wevent_booth_order_progress').remove();
                 const boothCategoryId = this.el.querySelector('input[name=booth_category_id]').value;
-                $form.replaceWith(renderToElement('event_booth_registration_complete', {
-                    'booth_category_id': boothCategoryId,
-                    'event_id': this.eventId,
-                    'event_name': jsonResponse.event_name,
-                    'contact': jsonResponse.contact,
-                }));
+                formEl.replaceWith(
```

Comment:
This raises a security issue.
Could we just add the new content and remove `formEl` instead to avoid it ?

Reply:
```js
                const newContent = renderToElement("event_booth_registration_complete", {
                    booth_category_id: boothCategoryId,
                    event_id: this.eventId,
                    event_name: jsonResponse.event_name,
                    contact: jsonResponse.contact,
                });
                formEl.insertAfter(newContent);
                formEl.remove();
```

Some thing like this ?

Reply:
varibale name newContent => `boothRegistrationCompleteForm` ?

Reply:
@bso-odoo Done

Reply:
:sweat_smile: @bso-odoo its done !